src/shogun/machine/Machine.h

```diff
@@ -237,6 +237,25 @@ class Machine : public StoppableSGObject
 			return true;
 		}
 
+		/** train classifier
+		 *
+		 * @param feat training data
+		 * @param lab training labels
+		 *
+		 * @return pointer to current Machine object
+		 */
+		std::shared_ptr<Machine>
+		fit(const std::shared_ptr<Features>& feat,
+		    const std::shared_ptr<Labels>& lab);
+
+		/** Perform classification on an array of test vectors X
```

Comment:
same here, it might be a regression task. I would say use the same description as in the old methods